libs/sdk-ui-charts/src/highcharts/adapter/chartPlugins.ts

```diff
@@ -7,35 +7,8 @@ import { groupCategoriesWrapper } from "./plugins/group-categories-wrapper";
 import { renderBubbles } from "./plugins/renderBubbles";
 import { adjustTickAmount } from "./plugins/adjustTickAmount";
 
-const extendRenderStackTotals = (Highcharts: any) => {
-    Highcharts.wrap(Highcharts.Axis.prototype, "renderStackTotals", function (proceed: any) {
-        // eslint-disable-next-line @typescript-eslint/no-this-alias
-        const axis = this;
-        const { chart, stackTotalGroup } = axis;
-        const { renderer } = chart;
-        /* We override renderStackTotals method to render "stack-labels" directly with desired
-         * visibility to prevent blinking of data labels while resizing. In Highcharts it's
-         * by default:
-         *     visibility: VISIBLE,
-         */
-        const defaultVisibility = chart.userOptions.stackLabelsVisibility || "visible";
```

Comment:
see the code setting `stackLabelsVisibility` the new code sets opacity of whole group so this code seems no longer needed. Anyway `renderStackTotals` is no longer part of `Highcharts.Axis.prototype` - it's one level deeper, under `stacking` which makes it unaccessible this way.